qa/rpc-tests/cfund-paymentrequest-extract-funds.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Navcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```

Comment:
Should we add a note in here about what bug this solves?
I understand that you can figure it out by reading the test itself but it gives a bit more context other that "extract funds".